src/chainlit/message.py

```diff
@@ -201,9 +201,10 @@ async def send(self):
         id = await super().send()
 
         action_coros = [action.send(for_id=str(id)) for action in self.actions]
```

Comment:
I understand the issue. This is a performance loss since we will sequentially wait to send (and persist, including files on s3). We can do that for now and enhance that later